sysutils/pkg/Portfile

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        freebsd pkg 1.10.4
+github.tarball_from releases
+categories          sysutils archivers
+platforms           darwin
+license             BSD
+maintainers         Email: ato.ms:apowers
```

Comment:
The text "`Email: `" is not valid and should be removed. Also please add `openmaintainer`.

Reply:
Fixed in latest commit

Reply:
The [best](https://guide.macports.org/#reference.keywords.maintainers) would be to include both GitHub username and email, so you can also be notified when pull requests are opened.

```
maintainers	{@apowers313 ato.ms:apowers} \
                openmaintainer
```

Reply:
Added in latest commit